backend/db/migrate/20210625092530_create_answers_options.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAnswersOptions < ActiveRecord::Migration[6.1]
```

Comment:
please add it to the class diagram image

Reply:
I think we should add a lot of classes to our ERD I will make a new issue for that

Reply:
agree!